invenio_curations/services/components.py

```diff
@@ -99,18 +99,7 @@ def update_draft(self, identity, data=None, record=None, errors=None):
             expand=True,
         )
 
-        # Inform user to create a curation request
         if not request:
-            errors.append(
-                {
-                    "field": "custom_fields.rdm-curation",
-                    "messages": [
-                        _(
-                            "Missing curation request. Please create a curation request, if the record is ready to be published."
-                        )
-                    ],
-                }
-            )
```

Comment:
we've removed the appended error in our customizations; not generating it in the first place (i.e. this change) is of course superior